### PR TITLE
First pass at read preference

### PR DESCRIPTION
**Pull request by acinader**
1. just getting started, will need to be adjusted when we make client accessible
2. are there more examples I should add??

### CODE REVIEW

**Comment by acinader** [comment]
@davimacedo please take a look.  anything we can add here??

**Comment by davimacedo** [comment]
I think it is only missing the includeReadPreference option

File: assets/js/bundle.js [hunk 1]
```diff
@@ -46,7 +46,7 @@
 /************************************************************************/
 /******/ ([
 /* 0 */
-/***/ function(module, exports, __webpack_require__) {
+/***/ (function(module, exports, __webpack_require__) {
```
Comment (flovilmart) [inline assets/js/bundle.js]: not sure this should be committed.

**Review by flovilmart — CHANGES_REQUESTED** [review]

**Comment by acinader** [comment]
@flovilmart

**Review by flovilmart — APPROVED** [review]

**Pull request merged**